File: Seclection sort/selection.js
function selection_sort(arr) {
    for (let i = 0; i < arr.length - 1; i++) {
        const element = arr[i];
        let min_idx = i;
        for (let j = 0; j < arr.length; j++) {
            if (arr[j] > arr[min_idx]) {
                min_idx = j
            };
        };
        let temp = arr[i];
        arr[i] = arr[min_idx];
        arr[min_idx] = temp;
        console.log(element);
    };
};
const arr = [20, 11, 13, 14, 12, 15, 9];
selection_sort(arr);

File: Seclection sort/selection.py
def selection_sort(arr):
    for i in range(len(arr)):
        # print(i)
        main_idx = i  # [3]

        for j in range(i + 1, len(arr)):
            if arr[j] > arr[main_idx]:
                main_idx = j
    print(main_idx)


arr = [3, 2, 1, 4, 6, 9, 7]
selection_sort(arr)
